README.md

```diff
@@ -55,6 +55,15 @@ Paimon tracks issues in GitHub and prefers to receive contributions as pull requ
 
 <b style="color:red">Please make sure you are subscribed to the mailing list you are posting to!</b> If you are not subscribed to the mailing list, your message will either be rejected (dev@ list) or you won't receive the response (user@ list).
 
+## Slack
+
+You can join the Paimon community on Slack. Paimon channel is in ASF Slack workspace.
```

Comment:
Adds the link of ASF Slack workspace to help users to join.

Reply:
> Adds the link of ASF Slack workspace to help users to join.

I added a link for join Slack

[![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://bit.ly/3KYbORi)

Reply:
@liugddx, IMO, you could add a commit in this pull request. Meanwhile, line 60 could also add a link.